q2_dada2/_denoise.py

```diff
@@ -266,15 +196,26 @@ def denoise_paired(demultiplexed_seqs: SingleLanePerSamplePairedEndFastqDirFmt,
             elif 'R2_001.fastq' in rp.name:
                 qiime2.util.duplicate(fp, os.path.join(tmp_reverse, rp.name))
 
-        cmd = ['run_dada_paired.R',
-               tmp_forward, tmp_reverse, biom_fp, track_fp, filt_forward,
-               filt_reverse,
-               str(trunc_len_f), str(trunc_len_r),
-               str(trim_left_f), str(trim_left_r),
-               str(max_ee_f), str(max_ee_r), str(trunc_q),
-               str(min_overlap), str(pooling_method),
-               str(chimera_method), str(min_fold_parent_over_abundance),
-               str(n_threads), str(n_reads_learn)]
+        cmd = ['run_dada_single_paired_ccs.R',
+               '--input_directory', tmp_forward,
+               '--input_directory_reverse', tmp_reverse,
+               '--output_path', biom_fp,
+               '--output_track', track_fp,
+               '--filtered_directory', filt_forward,
+               '--filtered_directory_reverse', filt_reverse,
+               '--tuncation_length', str(trunc_len_f),
```

Comment:
```suggestion
               '--truncation_length', str(trunc_len_f),
```

Reply:
Same comment in this file wrt `tuncation_length` typo - should all be replaced with `truncation_length`.